Update handbook (#584)
foundersandcoders/coursebook: src/course/handbook/user-research.md

> # User Research & Usability Testing

Reviewer: I do wonder if this should be in the design sprint section rather than the Handbook. Since they won't need it until ~week 7

Reviewer: Yeah handbook is mostly meant to be generic stuff they'll need all the time, rather than week-specific resources. For the main curriculum everything just gets dumped in "workshops" and linked in the schedule.

Author: <img width="486" alt="Screenshot 2022-02-16 at 10 59 54" src="https://user-images.githubusercontent.com/42008435/154251114-ddfbeebd-287d-4b67-8637-ef26ad7ba1bd.png">
would this top bar in the design week schedule be a good place to list it @oliverjam ?

Reviewer: I guess we could put this type of stuff in "Resources". I have historically just put external links in there, but no reason it can't be for our own stuff too

Reviewer 2: I suggested to Reuben yesterday that we make a 'documentation' folder within projects and stick all the docs in there - linked from the schedule

Author: this was proving tricky with the navigation (since the schedule/learnings/spikes  top bar uses relative navigation instead of full links)

When I added a file inside a nested documentation folder, clicking the top bar links after navigating to `/in-house-design/documentation/user-research` wouldn't work as they would now look for `/in-house-design/documentation/projects` instead of `/in-house-design/projects` 

<img width="248" alt="Screenshot 2022-02-17 at 10 33 41" src="https://user-images.githubusercontent.com/42008435/154458107-9404135e-a1cf-4ff1-8d6c-04c661e0c2ce.png">

Since the top bar is shared across weeks I've just added the `user-research.md` document to the `in-house-design` folder and linked it in the resources file and also on the schedule.

<img width="311" alt="Screenshot 2022-02-17 at 10 36 37" src="https://user-images.githubusercontent.com/42008435/154458612-45e60f38-a9b8-440c-9359-f5fee3aea76c.png">

<img width="207" alt="Screenshot 2022-02-17 at 10 36 45" src="https://user-images.githubusercontent.com/42008435/154458726-d3b69389-dac8-459b-a946-a8ea28bedbcf.png">

## Usability Testing

**[Usability testing](https://designsprintkit.withgoogle.com/methodology/phase6-validate/usability-study_1)** is a process for getting feedback from real people in order to inform the decisions we make on designing and building a digital product.

> - 'User Testing' and 'Usability Testing' refer to the same thing. The phrase ‘usability’ encourages us to remember it is not the user who we’re testing but the usability of our app or product. Do _not_ blame the user, instead take the opportunity to rethink your app.

Author: "Do take the opportunity to rethink your app, and take all criticism constructively!"

- You will be conducting an interview where you will ask the user to navigate through your Figma prototype.
- Do not give them too much instruction. Introduce them to the project idea, give them a few tasks to complete, and observe how they interact with the prototype.
- Their feedback will give you an idea of how someone unfamiliar with the project idea interacts with the app.

> - This will inform your decisions on which features to prioritise when building.

Reviewer 2: ```suggestion
- Feedback from testing will help inform your decisions on which features to prioritise when building.
```

## Process

During the in-house project design sprint (week 7) you will be conducting both
user research and usability testing.

> The Tech for Better product owners will conduct their own user research, but you will be conducting a usability testing session for your Tech for Better projects.

Reviewer 2: ```suggestion
The Tech for Better Product Owners will conduct their own user research, and you will be conducting usability testing with their user group.
```

- Check the design week schedules for exact timings:

>   [Student Project Schedule ](../course/syllabus/projects/in-house-design/schedule/)

Reviewer 2: ```suggestion
  [In-House Project Schedule ](../course/syllabus/projects/in-house-design/schedule/)
```

or 'In-house', not sure which we've used elsewhere
